### PR TITLE
added splash wrappers

### PR DESCRIPTION
**Pull request by wissog**
some prep for likutim

### CODE REVIEW

File: src/components/shared/WipErr/WipErr.js [hunk 1]
```diff
@@ -5,17 +5,29 @@ import * as shapes from '../../shapes';
 import { formatError } from '../../../helpers/utils';
 import { ErrorSplash, FrownSplash, LoadingSplash } from '../Splash/Splash';
 
+const errorStatusNotFound = err => (err.response && err.response.status === 404);
```
Comment (gshilin) [inline src/components/shared/WipErr/WipErr.js:8]: err?.response?.status === 404

**Review by gshilin — CHANGES_REQUESTED** [review]

**Comment by wissog** [comment]
i kept it as it was
בברכה,
גבריאל ויסוצקי
נייד: 054-5606-798

File: src/components/shared/WipErr/WipErr.js (hunk 1 above)
Comment (gshilin) [reply src/components/shared/WipErr/WipErr.js:8]: It was "old style". If you're already here and you're changing **everything** to be "new style", please, change it too...

**Comment by wissog** [comment]
it's an internal method that never accept an empty err...
בברכה,
גבריאל ויסוצקי
נייד: 054-5606-798

**Comment by gshilin** [comment]
Then `err.response?.status === 404` would be enough

File: src/components/shared/WipErr/WipErr.js (hunk 1 above)
Comment (wissog) [reply src/components/shared/WipErr/WipErr.js:8]: interesting that eslint misses it... 
but found a plugin that we may add for this: https://www.npmjs.com/package/eslint-plugin-prefer-optional-chaining
what do you think?

**Review by gshilin — APPROVED** [review]

**Pull request merged**